hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/handle/HandleWorkflowTest.java

```diff
@@ -89,14 +104,21 @@ class HandleWorkflowTest extends AppTestBase {
     private static final PreHandleResult DUE_DILIGENCE_RESULT = PreHandleResult.nodeDueDiligenceFailure(
             NODE_1.nodeAccountID(), ResponseCodeEnum.INVALID_TRANSACTION, new TransactionScenarioBuilder().txInfo());
 
-    private static PreHandleResult createPreHandleResult(@NonNull Status status, @NonNull ResponseCodeEnum code) {
+    private static PreHandleResult createPreHandleResult(
+            @NonNull final Status status, @NonNull final ResponseCodeEnum code) {
+        final var transactionId = TransactionID.newBuilder()
+                .accountID(AccountID.newBuilder().accountNum(1002L).build())
+                .build();
+
         final var key = ALICE.account().keyOrThrow();
         return new PreHandleResult(
                 ALICE.accountID(),
                 key,
                 status,
                 code,
-                new TransactionScenarioBuilder().txInfo(),
+                new TransactionScenarioBuilder()
+                        .withTransactionID(transactionId)
```

Comment:
This should not be necessary. There was a bug in `SingleTransactionRecordBuilder`, which is fixed now.